specs/EmergencyAuthCodeEligibleOfficers.yml

```diff
@@ -0,0 +1,223 @@
+openapi: '3.0.2'
+info:
+  title: Companies House Internal Oracle Query API
+  version: '1.0'
+servers:
+  - url: http://localhost:18XXX
+tags:
+  - name: emergency-auth-code-officers
+    description: company officers eligible for emergency auth code delivery
+paths:
+  /emergency-auth-code/company/{company_number}/eligible-officers:
+    parameters:
+      - $ref: '#/components/parameters/companyNumber'
+    get:
+      tags:
+        - emergency-auth-code-officers
+      operationId: listCompanyOfficers
+      summary: Get a list of the companies officers that are eligible for emergency auth code delivery
+      responses:
+        '200':
+          description: A list of eligible officers
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/companyOfficers'
+        '401':
+          description: Unauthorised
+        '404':
+          description: No eligible officers found
+  /emergency-auth-code/company/{company_number}/eligible-officers/{officer_id}:
+    parameters:
+      - $ref: '#/components/parameters/companyNumber'
+      - $ref: '#/components/parameters/officerId'
+    get:
+      tags:
+        - emergency-auth-code-officers
+      operationId: getCompanyOfficer
+      summary: Get a specific company officer that is eligible for emergency auth code delivery
+      responses:
+        '200':
+          description: An eligible officer
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/companyOfficer'
+        '401':
+          description: Unauthorised
+        '404':
+          description: Not found
+components:
+  schemas:
+    companyOfficer:
+      type: object
+      required:
+        - id
+        - name
+        - officer_role
```

Comment:
Seen as we're only returning directors this might be redundant

Reply:
This may be different for different company types so included it in to make it possible to extend. Though I imagine MVP would only deal with the simple case of "director"